### DISCUSSION TITLE
Dashboard says not connected?

### DISCUSSION THREAD

**Discussion by emdizzle** [tagyoureit/nodejs-poolController#1041] [App and O/S Config Questions]
Hi all:

Trying to get this up and running. I have everything installing in a pi 4, able to see messages in "/messageManager.html" in the dashboard. However, my main dashboard says "Unknown model", "Not connected". It is definitely getting data, but seems to be missing something. My server shows up when I search in settings/connections... not sure what I'm doing wrong. Any thoughts? Thanks!

**Comment by emdizzle** [comment]
This is the startup results. I'm wondering if the "waiting for installed modules" never happens.

![pool](https://github.com/user-attachments/assets/24573e67-7292-4a14-9428-97c8c257298d)

**Reply by emdizzle** [reply]
A couple of other notes, I replaced my 485 adapter with a new one that is recommended.

https://www.amazon.com/gp/product/B081MB6PN2/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&psc=1

I noticed on the LED lights that I am not getting any SEND/TRANSMIT activity. I am receiving data. Everything is wired up correctly. Additionally, I did have a Mobile touch connected to this system until recently, it stopped communicating. My panel works fine manually. I'm wondering if the lack of a controller on the system is causing issues. In my dashboard under controller/model, there is nothing listed.

**Reply by tagyoureit** [reply]
It looks like the packets are coming through correctly.  What model/software version of Intellicenter are you on?  1.064 was the last version that was supported by this app.  Any versions after that would need to have upgrades to the software.

**Reply by emdizzle** [reply]
I'm not sure what version, I have the IntelliTouch i9 with the personality board and the Mobi 2 which suddenly stopped working. Not really a way for me to see the firmware version. Unfortunately, I think my only course of action is to upgrade to the newer IntelliCenter.

**Reply by tagyoureit** [reply]
Very strange that you have Intellitouch but the all is picking up an Intellicenter.  Can you try stopping the all, deleting the two data files in ./data (poolConfig and poolState) and start the app again?
